examples/with-bun-react/README.md

```
@@ -0,0 +1,40 @@
+
+## Environment Variables
+
+This example includes a section that displays the `BUN_PUBLIC_MY_VALUE` environment variable. To set it up:
+
+1. Copy the example environment file:
+   ```bash
+   cp .env.example .env
+   ```
+
+2. Edit `.env` and set your `BUN_PUBLIC_MY_VALUE`:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)